### ISSUE TITLE
Issue : unknown footsteps sound

### ISSUE THREAD

**Issue by thiblizz** [emilyinure/solace-csgo#22]
@emilyinure, I keep hearing random footsteps when spectating someone or even when playing near some people while nobody is even walking or running.

**Comment by emilyinure** [comment]
Yes I've noticed this a lot before, never looked into but but I'll try to find the issue
